Src/ReadGps.c

```diff
@@ -7,14 +7,90 @@
 #include "Data.h"
 
 static int READ_GPS_PERIOD = 1000;
+static int CMD_TIMEOUT = 1500;
+
+static const uint8_t GPS_SET_ANTENNA[] = {0xB5, 0x62, 0x06, 0x13, 0x04, 0x00, 0x1B, 0x00, 0x8B, 0xA9}; // CFG-ANT
+static const uint8_t GPS_SET_DATUM[] = {0xB5, 0x62, 0x06, 0x06, 0x02, 0x00, 0x00, 0x00}; // CFG-DAT
+static const uint8_t GPS_SETUP_UART[];  //TODO: setup configuration for UART [CFG-PRT]
+static const uint8_t GPS_SET_OUTPUTRATE[] = {0xB5, 0x62, 0x06, 0x08, 0x06, 0x00, 0xFA, 0x00, 0x01, 0x00, 0x01, 0x00}; // CFG-RATE
```

Comment:
Should be like `GPS_SET_OUTPUT_RATE`

Reply:
One of the datasets that comes out of the NMEA string is UTC time which could be useful to log.

Reply:
See https://ukhas.org.uk/guides:ublox6#navigation_data

Reply:
Yea, that is on the list of things to log.